frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa.obcTest.scenario.spec.ts

```diff
@@ -0,0 +1,284 @@
+import {
+  BOUND,
+  MINUTE,
+  OBC_NAME,
+  OBC_RESOURCE_PATH,
+  OBC_RESOURCE_PATH,
+  OBC_STORAGE_CLASS,
+  OBC_STORAGE_CLASS_EXACT,
+  SECOND,
+} from '../../../noobaa-storage-plugin/integration-tests/utils/consts';
+import {
+  testDeployment,
+  deployment,
+} from '../../../noobaa-storage-plugin/integration-tests/mocks/deploymentData';
+import { testName } from '../../../integration-tests-cypress/support/index';
+import { projectNameSpace } from '../../../dev-console/integration-tests/support/pages/app';
+
+const MASK = '•••••';
+const NOOBAA_LABEL = 'app=noobaa';
+const NO_ANNOTATIONS = '0 annotations';
+
+const appHost = `${process.env.BRIDGE_BASE_ADDRESS || 'http://localhost:9000'}${(
+  process.env.BRIDGE_BASE_PATH || '/'
+).replace(/\/$/, '')}`;
+
+class CreateOBCHandler {
+  name: string;
+
+  namespace: string;
+
+  storageclass: string;
+
+  bucketclass: string;
+
+  constructor(name: string, namespace: string, storageclass: string, bucketclass: string) {
+    this.name = name;
+    this.namespace = namespace;
+    this.storageclass = storageclass;
+    this.bucketclass = bucketclass;
+  }
+
+  createBucketClaim() {
+    cy.clickNavLink(['Storage', 'Object Bucket Claims'], { timeout: 2 * MINUTE });
+    projectNameSpace.selectOrCreateProject(this.namespace);
+    cy.clickNavLink(['Storage', 'Object Bucket Claims']);
+    cy.get('[data-test-id="namespace-bar-dropdown"] div div button').click();
+    cy.contains(this.namespace);
+    cy.contains('Create Object Bucket Claim', { timeout: 6 * SECOND })
+      .should('be.visible')
+      .click();
+    cy.get('#obc-name').type(this.name);
+    cy.get('div.co-storage-class-dropdown div button')
+      .should('be.visible')
+      .click();
+    cy.contains('openshift-storage.noobaa.io').click();
+    if (this.bucketclass) {
+      cy.get('div.nb-create-obc__bc-dropdown div button')
+        .should('be.visible')
+        .click()
+        .contains(this.bucketclass)
+        .click();
+    }
+    cy.contains('button', 'Create').click();
+  }
+
+  verifySecretDataIsAvailable() {
+    cy.contains('Reveal Values').should(($el) => {
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      expect($el).to.exist;
```

Comment:
```suggestion
      expect($el).to.exist;
```
I don't think this would work. This is a chai assertion. I don't think we support chai in console.

Reply:
@bipuladh Cypress supports Chai assertions internally (https://docs.cypress.io/guides/references/assertions.html#Chai). Also as you can see below, the tests pass locally.

![image](https://user-images.githubusercontent.com/33557095/107183742-5649f400-6a05-11eb-965a-5fc1437b9a6c.png)

Reply:
Although Cypress supports it, https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/8061/pull-ci-openshift-console-master-frontend/1358690371428683776#1:build-log.txt%3A1555
we have no typescript support for it in our console, which is the reason it's failing `frontend` tests.
Please use jest assertions.